Tasks/AzureWebPowerShellDeployment/Publish-AzureWebDeployment.ps1

```diff
@@ -63,12 +63,16 @@ if($WebSiteLocation)
     if ($Slot)
     {
         Write-Host "Get-AzureWebSite -Name $WebSiteName -Slot $Slot -ErrorAction SilentlyContinue"
-        $azureWebSite = Get-AzureWebSite -Name $WebSiteName -Slot $Slot -ErrorAction SilentlyContinue
+        $azureWebSite = Get-AzureWebSite -Name $WebSiteName -Slot $Slot -ErrorAction SilentlyContinue -ErrorVariable azureWebSiteError
```

Comment:
Looks good. You may want to initialize $azureWebSiteError to $null before you call the command to make sure you're not reading any variable already set from an outer scope. May not be feasible based on how this script is called. Your call.